runner/runner.go

```diff
@@ -426,10 +436,12 @@ func (r *Runner) RunEnumeration() {
 			for _, p := range r.options.requestURIs {
 				scanopts := r.scanopts.Clone()
 				scanopts.RequestURI = p
+				r.ratelimiter.Take()
 				r.process(t, &wg, r.hp, protocol, scanopts, output)
 				reqs++
 			}
 		} else {
+			r.ratelimiter.Take()
```

Comment:
I think we can remove it here as it's already called in the `r.process` function (line 467)

Reply:
I ran some  tests removing this line but RPS seems to exceed the rate limit.